### PR TITLE
removed link from prototyping extension activity

### PR DESCRIPTION
**Pull request by vyaspranjal33**
removed link from prototyping extension activity as its referring to WebVR activities which isn't related to Prototyping your first browser extension

### CODE REVIEW

**Comment by MichaelKohler** [comment]
Thanks for your efforts. I don't think removing the link is what we want here. Like that we end up with no link at all. I think the real question here is what exactly this should link to.

I also see the following wrong links:
* Introduction to browser extensions --> should link to https://mozillacampusclubs.github.io/teach-how-to-build-addons/
* Prototyping your first browser extension --> should link to the same as the I am ready button

@bacharakis any thoughts?

**Comment by vyaspranjal33** [comment]
@MichaelKohler  i removed Prototyping your first browser extension  bcoz i don't found any relevant resource to point

**Comment by vyaspranjal33** [comment]
we can use any of the links or can use both @MichaelKohler what's your thoughts ?
Introduction to browser extensions -->https://developer.mozilla.org/en-US/Add-ons/WebExtensions 
 
OR 
Introduction to browser extensions --> https://developer.chrome.com/extensions

**Comment by vyaspranjal33** [comment]
Links has been updated

File: _pages/activities.html [hunk 1]
```diff
@@ -121,7 +121,7 @@ <h3 id="activities">Activities</h3>
 
     <div class="row activity">
       <div id="addons-activity" class="col-md-8">
-        <p><b><a href="https://mozillacampusclubs.github.io/Build-a-Virtual-World/">Introduction to browser extensions</a></b></p>
+        <p><b><a href="https://developer.mozilla.org/en-US/Add-ons/WebExtensions">Introduction to browser extensions</a></b></p>
```
Comment (bacharakis) [inline _pages/activities.html]: Hey can you please update the link with the activity's link as you did below?

**Comment by bacharakis** [comment]
Can you also please put back the button you removed in your first commit

**Comment by vyaspranjal33** [comment]
button has been putted

**Comment by vyaspranjal33** [comment]
@bacharakis anything needed to be worked ? on this PR or its ready to be merged

File: _pages/activities.html (hunk 1 above)
Comment (vyaspranjal33) [reply _pages/activities.html]: done

**Comment by bacharakis** [comment]
Hi @vyaspranjal33 ! Yes the button you added back points to the wrong activity. Can you please update the links, test them out before you push the changes and update the pull request again? Thank you and happy new year!

**Comment by vyaspranjal33** [comment]
hi @bacharakis  i've tested links

**Comment by bacharakis** [comment]
it's still missing the buttons but I will add them. Thank you for your work!

**Pull request merged**